src/modules/Channel/context/hooks/useInitialMessagesFetch.ts

```diff
@@ -129,6 +129,12 @@ function useInitialMessagesFetch(
         });
     }
   }, [channelUrl, userFilledMessageListQuery, initialTimeStamp]);
```

Comment:
Can we update `useEffect` to `useCallback`  and move it under the useCallback deps? (OR remove)

```
  /**
   * Note - useEffect(() => {}, [currentGroupChannel])
   * was buggy, that is why we did
   * const channelUrl = currentGroupChannel && currentGroupChannel.url;
   * useEffect(() => {}, [channelUrl])
   * Again, this hook is supposed to execute when currentGroupChannel changes
   * The 'channelUrl' here is not the same memory reference from Conversation.props
   */
```

Reply:
Ahh yeah couldn't even realized that the note was there 😅 I think it's better to keep just in case. c69a1a608b017b8eaf98731d5bafbd9c274d3fc8